### PR TITLE
Added collapsable components under Channels, Direct Messages, and Private Groups

### PR DESCRIPTION
**Pull request by bbrockman**
Added collapsable components under Channels, Direct Messages, and Private Groups
Added Kadira Debug
Added Boostrap - Included only required components for collapsible functionality
Added styles for collapsable components

### CODE REVIEW

**Comment by rwakida** [comment]
Verified it expands/collapses.  :+1: 

Suggestions:
1. Can we show the arrow only if there are items to collapse.  e.g. if user is not a channel member, then show "You aren't part of any channel yet" message and hide arrow.  
2. either move the arrow to be right-aligned with section's '+' or nudge to the right so that it's left-aligned with section title.

**Comment by bbrockman** [comment]
Good ideas.  

Just finished your first suggestion.  I'll take a look at your second.

**Comment by bbrockman** [comment]
Updated them both.  

Reid,
  Can you check out the updates to see if you agree with the changes.

**Comment by rwakida** [comment]
nice. :+1:

**Comment by douglasrapp** [comment]
I like it. Looks good to me.

File: client/lib/custom.bootstrap.mixins.import.less [hunk 1]
```diff
@@ -0,0 +1,1151 @@
...
+  @media (min-width: @screen-lg-min) {
+    left: percentage((@columns / @grid-columns));
+  }
+}
+.make-lg-column-pull(@columns) {
+  @media (min-width: @screen-lg-min) {
+    right: percentage((@columns / @grid-columns));
+  }
+}
+
```
Comment (douglasrapp) [inline client/lib/custom.bootstrap.mixins.import.less:1151]: I like how 99% of the code change in this pull request is adding giant bootstrap files ;)

Comment (bbrockman) [inline client/lib/custom.bootstrap.mixins.import.less:1151]: All so I can use the collapse functionality.

**Pull request merged**